app/models/models.py

```diff
@@ -0,0 +1,24 @@
+from pydantic.types import condecimal
+from sqlmodel import SQLModel, Field, UniqueConstraint
+from typing import Optional
+
+
+class User(SQLModel, table=True):
+    __table_args__ = (UniqueConstraint('email'),)
+    id: Optional[int] = Field(default=None, primary_key=True)
+    name: str
+    email: str
+    password: str
+
+
+class Account(SQLModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
+    price: condecimal(max_digits=10, decimal_places=3) = Field(default=0)
+    memo: str = None
```

Comment:
```suggestion
    memo: Optional[str] = None
```
으로 하거나 타입이 str니까 기본값을 빈 스트링으로 하면 좋을 것 같아요.
```python3
memo: str = ''
```

Reply:
수정 완료했습니다. 감사합니다!